app.rb

```diff
@@ -31,19 +32,74 @@
 config_overrides[:port] = ENV['APRUVE_PORT'] unless ENV['APRUVE_PORT'].nil?
 Apruve.configure(ENV['APRUVE_API_KEY'], apruve_environment, config_overrides)
 merchant_id = ENV['APRUVE_MERCHANT_ID']
+corporate_account_id = ENV['APRUVE_CORPORATE_ACCOUNT_ID']
 
 before '/webhook_notify' do
   request.body.rewind
   @webhook_body = request.body.read
 end
 
+get '/offline_order' do
+  erb :offline_orders
+end
+
+get '/customers' do
+  content_type :json
+  send_file File.expand_path('customers.json', settings.public_folder)
```

Comment:
One thing though is that through that endpoint, you still need an email, so we would still need a json, just to store emails correct? @tcochran317 at standup today you were talking about making the endpoint return a list of all of the corporate accounts if the email query parameter is not provided?